src/content/docs/en/reference/publish-to-npm.mdx

```diff
@@ -229,7 +229,7 @@ If you are extracting components from an existing project, you can even continue
 
 ## Testing your component
 
-Astro does not currently ship a test runner. This is something that we would like to tackle. _(If you are interested in helping out, [join us on Discord!](https://astro.build/chat/))_
+Astro does not currently ship a test runner. _(If you are interested in helping out tackling this, [join us on Discord!](https://astro.build/chat/))_
```

Comment:
```suggestion
Astro does not currently ship a test runner. _(If you are interested in helping out with this, [join us on Discord!](https://astro.build/chat/))_
```
I think this plainer language is more suitable, especially when we consider translating it. But, totally approve that docs is not the right place to say what we think we would like to do!